packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -101,10 +101,22 @@ export type Endpoints = {
     "GET",
     BlockArgs,
     {params: {block_id: string}},
-    BeaconBlockBody["attestations"],
+    AttestationListPhase0,
     ExecutionOptimisticAndFinalizedMeta
   >;
 
+  /**
+   * Get block attestations
+   * Retrieves attestation included in requested block.
+   */
+  getBlockAttestationsV2: Endpoint<
+    "GET",
+    BlockArgs,
+    {params: {block_id: string}},
+    AttestationList,
```

Comment:
same here, keep type coupled to block type

```suggestion
    BeaconBlockBody["attestations"],
```